### PR TITLE
feat: Use dropdown for speed control

### LINKED ISSUE #116: Replace speed slider with a dropdown
## Use Case

Requested by @lynwilhelm!
- Replace slider with a dropdown with the existing speed settings
- Add a input field in the dropdown for custom speed.
- Remove the divider between the speed control and the rest of the layout.

image

### PR DESCRIPTION
**Pull request by ShrimpCryptid**
Problem
=======
Closes #116 

Solution
========
- Replaces the speed slider with a dropdown component.

## Type of change
* New feature (non-breaking change which adds functionality)

Steps to Verify:
----------------
Click the preview link below and test a few different speed values.

Screenshots (optional):
-----------------------
image

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -722,18 +722,15 @@ function App(): ReactElement {
...
-                    fps={playbackFps}
-                    onChange={(fps) => {
-                      setPlaybackFps(fps);
-                      timeControls.setPlaybackFps(fps);
-                    }}
-                    disabled={disableTimeControlsUi}
-                  />
-                </div>
+              <div style={{ display: "flex", flexDirection: "row", flexGrow: 1, justifyContent: "flex-end" }}>
+                <PlaybackSpeedControl
```
Comment (toloudis) [inline src/App.tsx:726]: I always find it disappointing when you can't just drop in a component without N different wrapper elements.  I suppose it's all about careful positioning.

File: src/components/AppStyle.tsx [hunk 2]
```diff
@@ -244,6 +244,11 @@ export default function AppStyle(props: PropsWithChildren<AppStyleProps>): React
             Divider: {
               marginLG: 0,
             },
+            Modal: {
+              // Set z-index to 2000 here because Ant sets popups to 1050 by default, and modals to 1000.
+              zIndexBase: 2000,
+              zIndexPopupBase: 2000,
+            },
```
Comment (toloudis) [inline src/components/AppStyle.tsx:251]: also in #137 ?

Comment (ShrimpCryptid) [reply src/components/AppStyle.tsx:251]: Oops! I originally had these all in the same branch and was splitting them up for PRs, I missed this in my cleanup.

**Review by toloudis — APPROVED** [review]
LGTM

File: src/App.tsx (hunk 1 above)
Comment (ShrimpCryptid) [reply src/App.tsx:726]: This bit just aligns it with the left hand corner of the row for nice spacing-- it's totally optional but I found it looked better than just having it be part of the row.

At some point soon I should just make reusable div components for all of the repeated `flex` boilerplate I have 😋

**Review by kmitcham — APPROVED** [review]

**Pull request merged**